server/index.js

```diff
@@ -28,33 +28,33 @@ function register (server, options, next) {
   // mapreduce is required for `db.query()`
   PouchDB.plugin(require('pouchdb-mapreduce'))
 
-  if (!options.db.url) {
+  if (!options.PouchDB.url) {
```

Comment:
it’s more complicated than that. `options.db` is an object that you would pass into the `PouchDB` constructor, like this:

```js
var db = new PouchDB(options.db)
```

But instead of passing in options, we want to pass in the constructor itself, does that make sense?